cmd/ocm-backplane/healthCheck/health_check.go

```diff
@@ -0,0 +1,72 @@
+package healthcheck
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	checkVPN   bool
+	checkProxy bool
+)
+
+// HealthCheckCmd is the command for performing health checks
+var HealthCheckCmd = &cobra.Command{
+	Use:   "health-check",
+	Short: "Check VPN and Proxy connectivity",
+	Run: func(cmd *cobra.Command, args []string) {
+		// Always check VPN connectivity first
+		fmt.Println("Checking VPN connectivity...")
+		err := CheckVPNConnectivity()
+		if err != nil {
+			fmt.Println("VPN connectivity check failed:", err)
+			if checkProxy {
+				fmt.Println("Note: Proxy connectivity check requires VPN to be connected. Please ensure VPN is connected and try again.")
+			}
+			os.Exit(1)
+		} else {
+			fmt.Println("VPN connectivity check passed!")
+		}
+		fmt.Println()
```

Comment:
is this printing an empty line? 

just an idea, you may able to use go [text/template](https://pkg.go.dev/text/template) package to nicely arrange the output text

Reply:
Yeah, just printing an empty line for better output messages. Let me check `text/template` package to see how it can achieve that. Thanks :)